azure-pipelines.vrt-baseline.yml

```diff
@@ -0,0 +1,50 @@
+# Starter pipeline
+# Start with a minimal pipeline that you can customize to build and deploy your code.
+# Add steps that build, run tests, deploy, and more:
+# https://aka.ms/yaml
+
+trigger:
+  batch: true
+  branches:
+    include:
+      - master
+
+variables:
+  skipComponentGovernanceDetection: true
+
+pool:
+  vmImage: ubuntu-latest
```

Comment:
should this also use the `1ES-Host-Ubuntu` pool that we use for our other pipelines?

Reply:
Yes. Updated to 1ES-Host-Ubuntu. Since in test this did not had permission had set ubuntu-latest temporarily